### ISSUE TITLE
This would be perfect if you could create separate notes like macOS Notes app

### ISSUE THREAD

**Issue by u84six** [Kilian/fromscratch#106]
Is there any chance that you'll be adding a feature to create separate scratch notes and retrieve them individually? I love the simplicity of this. I use macOS Notes because all I really need to do is click, fast launch, create new note, start typing, and that's all I need to do. No saving of file names, nothing, just a local database of separate notes with titles. That's it! It would be great if Fromscratch had these features (for Windows):

- Create a new note (or you could say scratchpad)
- Auto save as you type (which you already do)
- Sidebar with all the separate scratchpads ordered by "last modified"
- A title that you can see in the sidebar
- Really fast launch (which the current version already has)

Future updates:
- Some basic text formatting (I like to underline, bold, italicize, change font size, and that's about it)
- Export to text file or something like that

I don't even care about syncing/cloud, because I like the security of it being a local database. And if I want to move it to another machine, the export feature would be good enough.

**Comment by Kilian** [comment]
Someone started building this a while ago so you could check that out: https://github.com/Kilian/fromscratch/pull/58 but I'm not sure if this is the direction I want to take FromScratch in (I don't want any "management" in FromScratch, like coming up with file names, or changing the order of notes or whatever)

**Comment by webbb82** [comment]
Ya separate notes would be it for me, Or if nothing else a clear button but sep notes is needed. Im using this for diff topics and putting these into a  bigger note app, but just my 2 cents great so far tho

**Comment by alexwilsonii** [comment]
I agree with all this. Just today I was looking for a simple, basic note taking app for Windows and found this. On Mac, I used to use Notational Velocity and it was nearly perfect in my opinion. I downloaded FromScratch and love it, but the fact that that I can't really generate a "list" or group of separate notes kills it for me. 

I really need a location on my drive where all these notes are saved and I can easily jump between them/search for one in particular by some keyword quickly from my keyboard. Look and feel, I LOVE this app, but usability for what I need, I ended up downloading ResophNotes which isn't nearly as aesthetically pleasing but fits what I need better. I reviewed #58 and it looks like it would give us what we need, but also looks to be dead given how stale it is.

Sidebar/separate notes and, ideally, resizing the window would make this app perfection.

**Comment by u84six** [comment]
> I agree with all this. Just today I was looking for a simple, basic note taking app for Windows and found this. On Mac, I used to use Notational Velocity and it was nearly perfect in my opinion. I downloaded FromScratch and love it, but the fact that that I can't really generate a "list" or group of separate notes kills it for me.
> 
> I really need a location on my drive where all these notes are saved and I can easily jump between them/search for one in particular by some keyword quickly from my keyboard. Look and feel, I LOVE this app, but usability for what I need, I ended up downloading ResophNotes which isn't nearly as aesthetically pleasing but fits what I need better. I reviewed #58 and it looks like it would give us what we need, but also looks to be dead given how stale it is.
> 
> Sidebar/separate notes and, ideally, resizing the window would make this app perfection.

Take a look at Standard Notes. It's probably what you're looking for, but they do charge for the full version. I find that the free version suffices for what I need. It's very much like macOS Notes

**Comment by u84six** [comment]
I'm not sure if this project is still going, but I've since switched to Joplin which has come a long way. Not sure why this is still open, so I'll close it
